src/Administrator.kt

```diff
@@ -1,40 +1,66 @@
-import java.util.* /** TODO: neeed to remove this because is not necessary */
+import User.Companion.EXIT_OPTION
 
 class Administrator(override val name: String, override val address: String, override val number: Int, override val dateOfBirth: Long, override val dateStarted: Long):User{
+    val DELETE_MOVIE_OPTION = "1"
+    val REGISTER_MOVIE_OPTION = "2"
+    val REGISTER_COPIES_OPTION = "3"
+    val REGISTER_ASSOCIATE_OPTION = "4"
+    val REGISTER_SALESMAN_OPTION = "5"
+
     override fun menu() {
         administratorMenu()
     }
 
+    /** TODO: Need to refactor these part maybe you need to think something design patter like factory or strategy **/
     private fun administratorMenu() {
         do {
-            println("1. eliminar pelicula")
-            println("2. registrar pelicula")
-            println("3. aumentar copias de pelicula")
-            println("4. registrar asociado")
-            println("5. registrar salesman")
-            println("0. salir")
+            println("$DELETE_MOVIE_OPTION. eliminar pelicula")
+            println("$REGISTER_MOVIE_OPTION. registrar pelicula")
+            println("$REGISTER_COPIES_OPTION. aumentar copias de pelicula")
+            println("$REGISTER_ASSOCIATE_OPTION. registrar asociado")
+            println("$REGISTER_SALESMAN_OPTION. registrar salesman")
+            println("$EXIT_OPTION. salir")
             println("ingrese una opcion: ")
             var op: String? = readLine()
```

Comment:
op is not a representaty name, how about option ? I think would be better isn't it?